### PR TITLE
updating to reflect new feature

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2838

@kaarins What do you think? :)

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md [hunk 1]
```diff
@@ -31,9 +31,7 @@ description: "Learn about changing your organization name in SharePoint URLs"
 
 # Change your SharePoint domain name
 
-Unfortunately, it isn't possible to change the SharePoint domain name for your organization in Microsoft 365. For example, if the name of your organization changes from "Contoso" to "Fabrikam," you can't change  `contoso.sharepoint.com`  to  `fabrikam.sharepoint.com`.
-  
-To use the domain name `fabrikam.sharepoint.com`, you would need to purchase a new Microsoft 365 subscription and move all email, files, and any other data you want to keep to the new subscription.
+It's now possible to change the SharePoint domain name, see [Rename your SharePoint domain](tenant-rename.md) for more information.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md:34]:
```suggestion
It's now possible to change the SharePoint domain name; see [Rename your SharePoint domain](tenant-rename.md) for more information.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@scanum @kaarins can we merge this? Thank you!

**Comment by kaarins** [comment]
We're not ready to add this to the main article just yet. :)

**Pull request closed without merging**